src/jsep.js

```diff
@@ -1,258 +1,427 @@
 //     JavaScript Expression Parser (JSEP) <%= version %>
 //     JSEP may be freely distributed under the MIT License
 //     https://ericsmekens.github.io/jsep/
+export class Jsep {
+	/**
+	 * @returns {string}
+	 */
+	static get version() {
+		// To be filled in by the template
+		return '<%= version %>';
+	}
+
+	/**
+	 * @returns {string}
+	 */
+	static toString() {
+		return 'JavaScript Expression Parser (JSEP) v' + Jsep.version;
+	};
+
+	// ==================== CONFIG ================================
+	/**
+	 * @method addUnaryOp
+	 * @param {string} op_name The name of the unary op to add
+	 * @returns {Jsep}
+	 */
+	static addUnaryOp(op_name) {
+		Jsep.max_unop_len = Math.max(op_name.length, Jsep.max_unop_len);
+		Jsep.unary_ops[op_name] = 1;
+		return Jsep;
+	}
+
+	/**
+	 * @method jsep.addBinaryOp
+	 * @param {string} op_name The name of the binary op to add
+	 * @param {number} precedence The precedence of the binary op (can be a float)
+	 * @returns {Jsep}
+	 */
+	static addBinaryOp(op_name, precedence) {
+		Jsep.max_binop_len = Math.max(op_name.length, Jsep.max_binop_len);
+		Jsep.binary_ops[op_name] = precedence;
```

Comment:
This is not for this PR, but I think eventually we may want to move towards a data structure that just works, vs having to use special-purpose mutation methods. E.g. we could make `binary_ops` an instance of a `BinaryOpMap` which would inherit from `Map` but intercept its methods to update the max length accordingly. Then people can just modify it like a regular map. What do you think?